backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -26,10 +26,29 @@ include::modules/oadp-backing-up-pvs-csi.adoc[leveloffset=+1]
 include::modules/oadp-backing-up-applications-restic.adoc[leveloffset=+1]
 include::modules/oadp-using-data-mover-for-csi-snapshots.adoc[leveloffset=+1]
 
+[id="oadp-12-data-mover-ceph"]
+== Using OADP 1.2 Data Mover with Ceph storage
+
+You can use OADP 1.2 Data Mover to backup and restore application data for a cluster that uses CephFS, a cluster that uses CephRBD, and for a cluster that uses both.
```

Comment:
I know it is pedantic but if we say for a cluster twice in three clauses, should we use it for the middle clause as well. IGNORE if you do not care :)